src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -1238,6 +1238,35 @@ internal bool HasRequiresLocationAttribute(EntityHandle token)
         }
 
 #nullable enable
+        /// <summary>
+        /// Indicates whether the first attribute should be prioritized over the second one.
+        /// Same order of priority as
+        ///   <see cref="TryGetDeprecatedOrExperimentalOrObsoleteAttribute(EntityHandle, IAttributeNamedArgumentDecoder, bool, bool)"/>
+        /// </summary>
+        internal static bool IsMoreImportantObsoleteKind(ObsoleteAttributeKind oldKind, ObsoleteAttributeKind newKind)
```

Comment:
>ObsoleteAttributeKind oldKind, ObsoleteAttributeKind newKind [](http://example.com/codeflow?start=57&length=60)

Consider naming parameters "first" and "second" for consistency with a doc comment above